OracleWebCenterSites/dockerfiles/12.2.1.4/Checksum

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
This is a checksum file. It should not have a copyright header. You just need line 10 and you should add a LF at the end of the file.